backend/application/syson-application/Dockerfile

```diff
@@ -1,10 +1,7 @@
-FROM node:18.7-alpine
-RUN adduser --disabled-password myuser
-RUN wget -O /etc/apk/keys/adoptium.rsa.pub https://packages.adoptium.net/artifactory/api/security/keypair/public/repositories/apk
-RUN echo 'https://packages.adoptium.net/artifactory/apk/alpine/main' >> /etc/apk/repositories
-RUN apk update && apk add temurin-17-jre
-COPY target/syson-application-*.jar ./
-RUN find ./ -name '*.jar' ! -name '*-sources.jar' -exec mv {} syson-application.jar \;
+FROM eclipse-temurin:17-jre-alpine
+RUN apk add --update-cache --no-cache nodejs npm && rm -rf /var/cache/apk/*
```

Comment:
Which version of node js will be installed? 18.7 is the targeted version

Reply:
Because I would prefer to have the same version as the one used by Sirius Web.
I know it's not related as this process is independent and syside only need node.js > 14.x but It would be more coherent to maintain the same version for all the stack.